### PR TITLE
Disable up/down ticker on strict `<NumberInput />`

### PR DESCRIPTION
**Pull request by frontned**
### 🔦 Summary

https://wix.slack.com/archives/C39FTGUBZ/p1598000521045500

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers
- [ ] 👨‍🎨 UX change is approved by the Design System UX
- 🔬 Change is tested
  - [x] Component

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5892/

File: src/NumberInput/NumberInput.js [hunk 1]
```diff
@@ -110,13 +110,17 @@ class NumberInput extends React.PureComponent {
...
   render() {
     // <Input/> should always be controlled. Therefore, not passing defaultValue to <Input/>.
-    const { suffix, defaultValue, ...props } = this.props;
+    const { suffix, defaultValue, strict, max, min, ...props } = this.props;
+    const value = this._getInputValueFromState();
 
     return (
       <Input
         {...props}
+        strict={strict}
```
Comment (EgoziE) [inline src/NumberInput/NumberInput.js:119]: strict is not a prop of Input

File: src/NumberInput/NumberInput.uni.driver.js [hunk 2]
```diff
@@ -13,5 +13,7 @@ export const numberInputDriverFactory = (base, body) => {
     clickOnIncrement: () => getTickerDriver().clickUp(),
     /** Click on ticker down */
     clickOnDecrement: () => getTickerDriver().clickDown(),
+    isDownDisabled: () => getTickerDriver().isDownDisabled(),
+    isUpDisabled: () => getTickerDriver().isUpDisabled(),
```
Comment (EgoziE) [inline src/NumberInput/NumberInput.uni.driver.js:17]: Please add:
1. comments to the functions
2. type definitions to `NumberInput.uni.driver.d` file

File: src/NumberInput/test/NumberInput.spec.js [hunk 3]
```diff
@@ -219,6 +219,7 @@ describe('NumberInput', () => {
     await driver.enterText('215');
     expect(onChange).toHaveBeenCalledWith(max);
     expect(await driver.getValue()).toEqual('10');
+    expect(await driver.isUpDisabled()).toBe(true);
```
Comment (EgoziE) [inline src/NumberInput/test/NumberInput.spec.js:222]: Please check both before and after.

File: src/NumberInput/test/NumberInput.spec.js [hunk 4]
```diff
@@ -230,6 +231,7 @@ describe('NumberInput', () => {
     await driver.enterText('-215');
     expect(onChange).toHaveBeenCalledWith(min);
     expect(await driver.getValue()).toEqual('0');
+    expect(await driver.isDownDisabled()).toBe(true);
```
Comment (EgoziE) [inline src/NumberInput/test/NumberInput.spec.js:234]: Please check both before and after.

**Review by EgoziE — CHANGES_REQUESTED** [review]

**Review by EgoziE — APPROVED** [review]

**Review by ZilvinasMaciulis — APPROVED** [review]

**Pull request merged**